packages/app/src/app/components/sandbox/SandboxList.js

```diff
@@ -100,6 +101,10 @@ export default ({ sandboxes, isCurrentUser, onDelete }: Props) => (
           </td>
           <td>{moment(s.insertedAt).format('ll')}</td>
           <td>{moment(s.updatedAt).format('ll')}</td>
+          <StatBody>
+            {getDefinition(s.template).Icon(35, 35) ||
```

Comment:
I would replace `{getDefinition(s.template).Icon(35, 35) || getDefinition().Icon(35, 35)}` with something like this:

At the beginning of the render body:

const Icon = getDefinition(s.template).Icon || getDefinition().Icon;

Reply:
I also thought this (I remembered this would be better after pushed my changes) but waited to see your opinion 😃

Reply:
Oh, but this is in stateless function. So I'll replace `(` with `{`